index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+
+    <nav id="navbar">
+      <a id="profile-link" href="https://codepen.io/pen?template=MJjpwO"
+         >My profile
+      </a>
+      <a class="links" href="#projects">Projects</a>	
+      <a class="links" href="#more">More</a>
```

Comment:
I recommend that these links be in an unordered list (ul), with each list (li) tag enclosing the anchors (a).

Reply:
- [ ] Instead of framing in this way, you can write instead "It is recommended that you wrap these links in `ul tag`, with `li tag` enclosing in `a tag` following the HTML best practices."
_You have not given the why for your review comment in this._